LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Blue Fire
+Copyright (c) 2021-2022 Blue Fire
```

Comment:
This shouldn't need to be a range afaik, but the first year of the publication for what it's covering, so 2021 should be fine.

Reply:
ok, changed it back